### DISCUSSION TITLE
how to disable Hyperlink?

### DISCUSSION THREAD

**Discussion by sdwhsrw** [sumatrapdfreader/sumatrapdf#4550] [Q&A]
I'm reading a very annoying pdf document. All professional words in it are set with Hyperlinks to the vocabulary in the Appendix. 
So I have to scroll back, once I click those words accidentally.  

Is there a choice in settings to disable all hyperlinks or any other way to solve this problem ?

**Comment by tactilis** [comment]
> So I have to scroll back, once I click those words accidentally.

There's no need to scroll.

Press Alt+Left Arrow to go back (just as you would do for Go Back in a web browser)

In SumatraPDF see Help > Keyboard Shortcuts

**Reply by GitHubRulesOK** [reply]
In pre-release you can switch the blue boundary indicator on via debug show links, to help avoid alighting on the blue touch paper (poor joke) but not disable the links.
![image](https://github.com/user-attachments/assets/702f9c7f-7f05-4f9e-ba9b-d9e4fa114480)

Restrictions can only be set for external hyperlinks  
>Opening a web browser after a click on a hyperlink
>Launching external PDF viewers, LaTeX source editors or media players

**Comment by kjk** [comment]
There is now: the next pre-release build (3.7) has a `DisableLinks` advanced setting plus a **Toggle Disable Links** command.

How to use it:

- `Ctrl + K` (command palette) → **Toggle Disable Links**. The setting is saved, so it survives a restart.
- Or set it directly in `Settings` → `Advanced Options...`: `DisableLinks = true`.

When it's on, document links are ignored: a click or double-click on one doesn't jump anywhere, hovering doesn't give the hand cursor or the link tooltip, and Tab / keyboard link following is off. So a document where every technical term links into an appendix can be read and selected without accidentally jumping.

This is different from the existing **Toggle Show Links**, which only draws a blue outline around links — it doesn't disable them.

To get a key for it, bind it in advanced settings:

```
Shortcuts [
    [
        Cmd = CmdToggleDisableLinks
        Key = Alt + L
    ]
]
```

`CmdToggleDisableLinks` also takes an optional state argument (`CmdToggleDisableLinks on` / `CmdToggleDisableLinks off`) if you want to force a state instead of flipping it.
